flink-formats/flink-avro-glue-schema-registry/src/main/java/org/apache/flink/formats/avro/glue/schema/registry/AvroGlueFormatOptions.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ package org.apache.flink.formats.avro.glue.schema.registry;
+
+import java.time.Duration;
+
+import com.amazonaws.services.schemaregistry.utils.AWSSchemaRegistryConstants;
+import com.amazonaws.services.schemaregistry.utils.AWSSchemaRegistryConstants.COMPRESSION;
+import com.amazonaws.services.schemaregistry.utils.AvroRecordType;
+
+import org.apache.flink.annotation.PublicEvolving;
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.ConfigOptions;
+
+import software.amazon.awssdk.services.glue.model.Compatibility;
+
+@PublicEvolving
+public class AvroGlueFormatOptions {
+    public static final String PREFIX = "schema-registry.";
+
+    public static final ConfigOption<String> AWS_REGION = ConfigOptions.key(PREFIX + AWSSchemaRegistryConstants.AWS_REGION)
+            .stringType().noDefaultValue().withDescription("AWS region");
+
+    public static final ConfigOption<String> REGISTRY_NAME = ConfigOptions.key(PREFIX + AWSSchemaRegistryConstants.REGISTRY_NAME)
+            .stringType().noDefaultValue().withDescription("Registry name");
+
+    public static final ConfigOption<String> RECORD_TYPE = ConfigOptions
+            .key(PREFIX + AWSSchemaRegistryConstants.AVRO_RECORD_TYPE).stringType()
+            .defaultValue(AvroRecordType.GENERIC_RECORD.getName()).withDescription("Record type");
+
+    public static final ConfigOption<String> SCHEMA_REGISTRY_SUBJECT = ConfigOptions.key(PREFIX + "subject").stringType()
```

Comment:
Same as above: can this be named to `SCHEMA_NAME`?

Reply:
done